back/config/schemas/settings.schema.json.erb

```diff
@@ -22,6 +22,7 @@
           "authentication_token_lifetime_in_days"
         ],
         "additionalProperties": false,
+        "pricing": true,
```

Comment:
This isn't technical an attribute on a JSON schema definition, but nothing breaks when we add it. Maybe there is something I need to do to allow it 'officially'